### DISCUSSION TITLE
Optional lock to prevent accidental deletion of important aliases

### DISCUSSION THREAD

**Discussion by Gorson84** [simple-login/app#926] [Feature Requests]
Hi,

It would be nice to have the ability to lock certain aliases to prevent accidental deletion.
Unlocking could be done via the normal password of the account

thx
Gorson

**Comment by strawrhu** [comment]
I completely agree. Deleting several alaises in a row can be nerve-wracking, for this very reason. (Sometimes I have a moment of terror after deleting the third or fourth, say, questioning whether or not I was paying sufficient attention or starting to zone out a bit and getting click-happy without paying enough attention.) I've often wished for this feature!

**Comment by vbc4ejyg** [comment]
+10

Create a "locked" toggle similar to the "pin" on an alias that requires account password to delete (somewhat of a "are you really sure" workflow).


This would avoid a too hastily click and delete for Proton aliases that can never be recover once it's deleted.  Key here is to make it harder to delete critical aliases that you really want to be sure you wanted deleted.  It could even show the alias notes in the "Are you very sure? As this alias can never be recovered" dialogue box.
